src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -198,14 +212,14 @@ export default class CaseAlterationTable extends React.Component<ICaseAlteration
         ];
 
         const geneSet : {[x: string]: OQLLineFilterOutput<AnnotatedExtendedAlteration>} = {};
-        this.props.oqls.forEach(oql => {
+        _.forEach(_.cloneDeep(this.props.oqls), (oql => {
```

Comment:
why clonedeep?

Reply:
I noticed that if I use forEach() to iterate the `this.props.oqls`. I will change the value of `oql_line`. So I used `_.cloneDeep` to remove the reference.